File: algo_questions/easy/two_number_sum/two_number_sum.ts
export function twoNumberSum(array: number[], targetSum: number) {
    // Write your code here.
    let result: number[] = [];
    for (let i: number = 0; i < array.length; i++) {
        for (let k: number = i + 1; k < array.length; k++) {
            if (array[i] + array[k] === targetSum) {
                result = [ array[i], array[k] ]
                return result
            } 
        }
    }
    return result;
  }

File: algo_questions/easy/two_number_sum/two_number_sum.swift
class Program {
  func twoNumberSum(_ array: inout [Int], _ targetSum: Int) -> [Int] {
    // Write your code here.
    var result: [Int] = []
    for (index, number) in array.enumerated() {
        // start at first number index
        for second_number in array[(index + 1)...] {
            if number + second_number == targetSum {
                result = [number, second_number]
                return result
            }
        }
    }
    return result
  }
}